packages/extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "extension",
-  "version": "3.35.7",
+  "version": "3.35.8",
```

Comment:
How did this change here? :jenkins-triggered:

Reply:
Target branch didn't have it, just update and now is fine ✅